libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1047,10 +1047,11 @@ void AP_TECS::_update_pitch(void)
     _last_pitch_dem = _pitch_dem;
 
     if (AP::logger().should_log(_log_bitmask)){
-        AP::logger().WriteStreaming("TEC2","TimeUS,PEW,EBD,EBE,EBDD,EBDE,EBDDT,Imin,Imax,I,KI,pmin,pmax",
-                                    "Qffffffffffff",
+        AP::logger().WriteStreaming("TEC2","TimeUS,PEW,KEW,EBD,EBE,EBDD,EBDE,EBDDT,Imin,Imax,I,KI,pmin,pmax",
```

Comment:
Not really your problem, but I find the log field names in this message extremely confusing. So bonus points for adding some documentation for this message.

Reply:
Fair point. I know I'll regret it if I don't. Added.

Reply:
Thank you!